README.md

```diff
@@ -1,7 +1,10 @@
 Pandas cookbook
 ===============
 
-[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/jvns/pandas-cookbook/master)
```

Comment:
Binder has fallen out of favor in the jupyter community generally because became very unreliable after google stopped supporting it